ctrl_dbus: Parallel build random fails

When compiling `ctrl_dbus` compiling will fail random if compiled with multiple threads.
This happens because it is not guaranteed that `baresipbus.h` is generated before `ctrl_dbus.c` is compiled.

Log:
```
[  117s] gcc -fmessage-length=0 -grecord-gcc-switches -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -g -fpie -I/usr/include/ffmpeg -DRELEASE -Wall -Wextra -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wbad-function-cast -Wsign-compare -Wnested-externs -Wshadow -Waggregate-return -Wcast-align -Wold-style-definition -Wdeclaration-after-statement -g -O3   -Wuninitialized -Wno-strict-aliasing -fPIC -DLINUX -DOS=\"linux\" -std=c11 -pedantic -DHAVE_ATOMIC -DARCH=\"x86_64\" -DUSE_OPENSSL -DUSE_TLS -DUSE_OPENSSL_DTLS -DUSE_DTLS -DUSE_OPENSSL_SRTP -DUSE_DTLS_SRTP -DUSE_ZLIB -DHAVE_PTHREAD -DHAVE_GETIFADDRS -DHAVE_STRERROR_R -DHAVE_GETOPT -DHAVE_INTTYPES_H -DHAVE_NET_ROUTE_H -DHAVE_SYS_SYSCTL_H -DHAVE_STDBOOL_H -DHAVE_INET6 -DHAVE_RESOLV -DHAVE_SYSLOG -DHAVE_FORK -DHAVE_INET_NTOP -DHAVE_PWD_H -DHAVE_POLL	 -DHAVE_INET_PTON -DHAVE_SELECT -DHAVE_SELECT_H -DHAVE_SETRLIMIT -DHAVE_SIGNAL -DHAVE_SYS_TIME_H -DHAVE_EPOLL -DHAVE_UNAME -DHAVE_UNISTD_H -DHAVE_STRINGS_H -DVERSION=\"1.1.0\" -DVER_MAJOR= -DVER_MINOR= -DVER_PATCH=  -I. -Iinclude -I/usr/include/re -I/usr/include/rem -DPREFIX=\"/usr/\" -DMOD_PATH=\"/usr/lib64/baresip/modules\" -DSHARE_PATH=\"/usr//share/baresip\" -pthread -isystem/usr/include/gio-unix-2.0 -isystem/usr/include/libmount -isystem/usr/include/blkid -isystem/usr/include/uuid -isystem/usr/include/glib-2.0 -isystem/usr/lib64/glib-2.0/include -Wno-unused-parameter -Wno-declaration-after-statement -Wno-pedantic -Wno-shorten-64-to-32 \
[  117s] 	-c modules/ctrl_dbus/ctrl_dbus.c -o build-x86_64/modules/ctrl_dbus/ctrl_dbus.o -MD -MF build-x86_64/modules/ctrl_dbus/ctrl_dbus.d -MT build-x86_64/modules/ctrl_dbus/ctrl_dbus.o
[  117s] modules/ctrl_dbus/ctrl_dbus.c:69:10: fatal error: baresipbus.h: No such file or directory
[  117s]  #include "baresipbus.h"
[  117s]           ^~~~~~~~~~~~~~
[  117s] compilation terminated.
[  117s] make: *** [mk/mod.mk:49: build-x86_64/modules/ctrl_dbus/ctrl_dbus.o] Error 1
```

--- a/modules/ctrl_dbus/module.mk
+++ b/modules/ctrl_dbus/module.mk
@@ -16,7 +16,10 @@ $(MOD)_CFLAGS	+= -Wno-unused-parameter -Wno-declaration-after-statement \
 
 $(MOD)_CCHECK_OPT	= -e baresipbus.h -e baresipbus.c
 
-modules/ctrl_dbus/baresipbus.h modules/ctrl_dbus/baresipbus.c: \
+modules/$(MOD)/baresipbus.o :	modules/$(MOD)/baresipbus.h
+modules/$(MOD)/ctrl_dbus.o :	modules/$(MOD)/baresipbus.h
+
+modules/$(MOD)/baresipbus.h modules/$(MOD)/baresipbus.c: \
 	modules/ctrl_dbus/com.github.Baresip.xml
 	@cd $(dir $@) && ./gen.sh
 

